build.xml

```diff
@@ -134,12 +134,15 @@
         <antcall target="clean"/>
     </target>
     
-    <target name="init" description="initializes the build environment, e.g., downloads saxon" unless="${docker}">
+    <target name="init" description="Initialize: checks if saxon and xerces are available, if not downloads them." unless="${docker}">
         <available property="xerces-available" file="${dir.lib.xerces}/${xerces.jar.file}"/>
+        <available property="xerces-available" classname="org.apache.xerces.jaxp.SAXParserFactoryImpl"/>
```

Comment:
Is it correct that this line addresses the same property name as the line before?

Can you briefly indicate why this additional check is needed (same for line 142 with Saxon)?

Reply:
Currently, the mei.classpath is composed of the jars available in the lib folder and the system java.classpath. The double check checks for the lib folder first – whenever a property in Apache Ant is first initialized with a value, it is immutable afterward – so the second check for the classpath availability checks if the java.classpath already contains the desired classes; if so, it is unnecessary to download the jars.